spec/controllers/datasets/update_spec.rb

```diff
@@ -305,7 +305,7 @@
         {
           title: "New file",
           description: "New file description",
-          file: "http://example.com/new-file.csv",
```

Comment:
Because the file key in the S3 uploader is no longer a URL, just the storage key. So I've removed the protocol and port which would have been added before.